### PR TITLE
Update django.po

### PR DESCRIPTION
**Pull request by mnp-fuf**

### CODE REVIEW

**Comment by joshblum** [comment]
Thanks @mnp-fuf!

File: registration/locale/pl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -129,7 +129,12 @@ msgid ""
 "next\n"
 "    %(expiration_days)s days:\n"
 "    "
-msgstr ""
+msgstr  ""
+"\n"
+"    Aby aktywować to konto, kliknij w poniższy link w ciągu "
+"next\n"
```
Comment (zapu) [inline registration/locale/pl/LC_MESSAGES/django.po:135]: "next"?

File: registration/locale/pl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -284,35 +298,35 @@ msgstr "hasło (ponownie)"
...
 
 #: templates/registration/password_reset_complete.html:4
 msgid "Password reset complete"
-msgstr ""
+msgstr "Hasło zresetowane"
 
 #: templates/registration/password_reset_complete.html:8
 msgid "Your password has been reset!"
-msgstr ""
+msgstr "Twoje hasło zostaało zresetowane!"
```
Comment (zapu) [inline registration/locale/pl/LC_MESSAGES/django.po:316]: "zost**aa**ło"

File: registration/locale/pl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -284,35 +298,35 @@ msgstr "hasło (ponownie)"
...
 
 #: templates/registration/password_reset_confirm.html:10
 msgid "Confirm password reset"
-msgstr ""
+msgstr "Potwierdź reset hasła"
 
 #: templates/registration/password_reset_confirm.html:14
 msgid "Enter your new password below to reset your password:"
-msgstr ""
+msgstr "Wprowaðx ponizej nowe hasło, by zresetować swoje hasło"
```
Comment (zapu) [inline registration/locale/pl/LC_MESSAGES/django.po:329]: "Wprowaðx" ?

File: registration/locale/pl/LC_MESSAGES/django.po (hunk 1 above)
Comment (mnp-fuf) [reply registration/locale/pl/LC_MESSAGES/django.po:135]: Fixed

File: registration/locale/pl/LC_MESSAGES/django.po (hunk 2 above)
Comment (mnp-fuf) [reply registration/locale/pl/LC_MESSAGES/django.po:316]: Fixed

File: registration/locale/pl/LC_MESSAGES/django.po (hunk 3 above)
Comment (mnp-fuf) [reply registration/locale/pl/LC_MESSAGES/django.po:329]: Fixed. Next time feel free to correct it yourself. It won't take you longer than commenting ;)

Comment (mnp-fuf) [reply registration/locale/pl/LC_MESSAGES/django.po:329]: Fixed.

**Pull request merged**